master/jobmaster/jobmanager.go

```diff
@@ -6,6 +6,7 @@ import (
 
 	"github.com/hanfei1991/microcosm/master/cluster"
 	"github.com/hanfei1991/microcosm/master/jobmaster/benchmark"
+	"github.com/hanfei1991/microcosm/master/jobmaster/system"
```

Comment:
jobmanager.go should be under the  master  folder ,since it is a part of server master ,not the job master

Reply:
addressed